aten/src/ATen/core/interned_strings.h

```diff
@@ -65,10 +65,10 @@ namespace c10 {
   _(prim, NumToTensor)             \
   _(prim, Uninitialized)           \
   _(prim, ImplicitTensorToNum)     \
-  _(prim, Bool)                    \
-  _(prim, Int)                     \
-  _(prim, Float)                   \
-  _(prim, str)                     \
+  _(aten, Bool)                    \
+  _(aten, Int)                     \
+  _(aten, Float)                   \
+  _(aten, Str)                     \
```

Comment:
Please don't  capitalize, it is suppose to match Python. Note that Bool/Int/Float have to be different because they would collide with C keywords